src/test/unit/specific_feature_tests/exec_toggles.js

```diff
@@ -318,11 +318,11 @@ test("IE unwraps partially subscript selection", function () {
     );
 });
 
-function doesntWrapAcrossRootContainers(command) {
-    var noChangeHtml = "<p>Foo</p><p>Bar</p>";
+function wrapsAcrossRootContainers(command) {
+    var tagName = getBrowserTagname(command);
     manipulationTestHelper({
-        startHtml: noChangeHtml,
-        prepareFunf: function (wymeditor) {
```

Comment:
`manipulationTestHelper` could do a validation on the object it receives, and if there are any keys missing, or unexpected keys passed it, it aborts.